gqlauth/models.py

```diff
@@ -58,16 +58,28 @@ def send(self, subject, template, context, recipient_list=None):
 
     def get_email_context(self, info: Info, path, action, **kwargs):
         token = get_token(self.user, action, **kwargs)
-        request = info.context.request
-        site = get_current_site(request)
+        is_channels = isinstance(info.context, dict)
+        if is_channels:
+            request = info.context["request"]
+            domain, port = request.headers["host"].split(":")
+            site_name = domain
+            protocol = request.consumer.scope["type"]
+        else:
+            request = info.context.request
+            site = get_current_site(request)
+            port = request.get_port()
+            site_name = site.name
+            domain = site.domain
+            protocol = ("https" if request.is_secure() else "http",)
```

Comment:
that's a tuple :upside_down_face:

Reply:
No it wasn't. but you what you wrote is a tuple
`("https" if request.is_secure() else "http",)`
`
```suggestion
            protocol = "https" if request.is_secure() else "http",
```

Reply:
It seems to have been automatically fixed by [pre-commit-ci[bot]. I'd like you to take a look at this bot.